windows/src/engine/keyman32/keyman32.cpp

```diff
@@ -838,7 +831,7 @@ void RefreshKeyboards(BOOL Initialising)
 
 void ReleaseKeyboards(BOOL Lock)
 {
-  OutputThreadDebugString("ReleaseKeyboards");
+  // OutputThreadDebugString("ReleaseKeyboards");
```

Comment:
Are you wanting to leave these here commented out so that you can turn it on for developer debuging later? Do we have a if #DEBUG macro we or function we could call that wraps the OutputThreadDebugString and NOPS if not build DEBUG?
I probably should know all this.